docker-entrypoint.sh

```diff
@@ -16,9 +16,10 @@ echo "INSTALLDIR=${INSTALLDIR}"
 
 pg_isready -d stopo_prod -h ${DBHOST} -p ${DBPORT} -U ${PGUSER}
 
-source rc_${DEPLOY_TARGET} && make production.ini development.ini
+cut -c8- ${DOCKER_ENV_FILE} > ${DOCKER_ENV_FILE}_no_export
```

Comment:
This could help to get rid of `dev_frankfurt.env` and `dev.env`, as only `rc_dev` and `rc_dev_frankfurt` will be sufficient